packages/lib-classifier/src/store/WorkflowStore/WorkflowStore.js

```diff
@@ -22,8 +22,6 @@ const WorkflowStore = types
         const validProjectReference = isValidReference(() => getRoot(self).projects?.active)
         if (validProjectReference) {
           self.reset()
```

Comment:
Should this reset move into the `selectWorkflow` action, so that we always reset the store before setting a new workflow?